### PR TITLE
WIP: more scaling fixes

### PR DESCRIPTION
**Pull request by willcl-ark**

### CODE REVIEW

File: src/services/fluentd.py [hunk 1]
```diff
@@ -21,9 +21,9 @@ def __init__(self, docker_network, config_dir):
             "command": ["/bin/sh", "-c", f"fluentd -c /fluentd/etc/{FLUENT_CONF}"],
             "healthcheck": {
                 "test": ["CMD", "/bin/bash", "-c", f"cat < /dev/null > /dev/tcp/localhost/{self.PORT}"],
-                "interval": "5s",           # Check every 5 seconds
+                "interval": "20s",           # Check every 5 seconds
                 "timeout": "1s",            # Give the check 1 second to complete
-                "start_period": "2s",       # Start checking after 2 seconds
+                "start_period": "5s",       # Start checking after 2 seconds
```
Comment (pinheadmz) [inline src/services/fluentd.py:26]: nit: comments dont match code.

File: src/graphs/default.graphml [hunk 2]
```diff
@@ -6,7 +6,7 @@
   <graph edgedefault="directed">
     <node id="0">
         <data key="version">25.0</data>
-        <data key="bitcoin_config">uacomment=w0</data>
+        <data key="bitcoin_config">uacomment=w0, debug=addrman</data>
```
Comment (pinheadmz) [inline src/graphs/default.graphml:9]: whats this for?

File: src/interfaces/docker_interface.py [hunk 3]
```diff
@@ -32,94 +32,67 @@
...
 class DockerInterface(ContainerInterface):
     def __init__(self, network: str, config_dir: Path) -> None:
         super().__init__(network, config_dir)
         self.client = docker.DockerClient = docker.from_env()
 
     @bubble_exception_str
     def build(self) -> bool:
-        command = ["docker", "compose", "build"]
-        try:
-            with subprocess.Popen(
```
Comment (pinheadmz) [inline src/interfaces/docker_interface.py:44]: I don't understand the commit message -- we *were* using subprocess already ...?

File: src/interfaces/docker_interface.py [hunk 4]
```diff
@@ -158,25 +131,28 @@ def get_messages(self, a_name: str, b_ipv4: str, bitcoin_network: str = "regtest
...
+        command = f"docker logs {container_name}"
+        args = shlex.split(command)
         matching_logs = []
-        for log_entry in log_stream:
-            log_entry_str = log_entry.decode('utf-8').strip()
-            if compiled_pattern.search(log_entry_str):
-                matching_logs.append(log_entry_str)
 
+        try:
+            with subprocess.Popen(
```
Comment (pinheadmz) [inline src/interfaces/docker_interface.py:141]: why didn't you use `run_subprocess()` here?

**Review by pinheadmz — COMMENTED** [review]
Getting there, Im gonna push some commits to this branch for your review @willcl-ark and bang on CI

File: src/graphs/default.graphml (hunk 2 above)
Comment (willcl-ark) [reply src/graphs/default.graphml:9]: Just to show how it's done, now that I set debug=0 as default

File: src/interfaces/docker_interface.py (hunk 4 above)
Comment (pinheadmz) [reply src/interfaces/docker_interface.py:141]: oh I see, because we're filtering results with `grep`

Comment (willcl-ark) [reply src/interfaces/docker_interface.py:141]: Sorry yeah. That's why. I think clever code could still merge the two, but figured it was an optimisation

**Comment by willcl-ark** [comment]
rebased on main

**Pull request closed without merging**